department-of-veterans-affairs/abd-vro
# JDBC prefix only for flyway db-init

## What was the problem?
Fixes #2309 

Associated tickets or Slack threads:
- #?

## How does this fix it?[^1]
- Changes the env var used by db-init to be consistent with other postgres url convention

## How to test this PR
- Deployed this to dev env and verified the changes


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
